sdks/go/pkg/beam/core/runtime/graphx/translate.go

```diff
@@ -682,6 +683,19 @@ func (m *marshaller) expandCrossLanguage(namedEdge NamedEdge) (string, error) {
 		EnvironmentId: m.addDefaultEnv(),
 	}
 
+	// Add the coders for output types in the marshaller even if expanded is nil
+	// to set the output coder request field in expansion request for python external transforms.
+	names := strings.Split(spec.Urn, ":")
+	if len(names) > 2 && names[2] == "python" {
```

Comment:
Does anything fail when we always provide output codings to the expansion services?
Ideally if Java doesn't use them, they should be ignored, and all the expansion outputs should be known Beam coders (rather than Java or Python or Go specific).

Reply:
That was with an earlier approach though which I made it simpler later in latest commit. Testing it locally for commit [f69475d](https://github.com/apache/beam/pull/23641/commits/f69475dc51aa4c9d4c43c0e089db5d8e254a5250)

Reply:
Confirmed that this error occurs in [f69475d](https://github.com/apache/beam/pull/23641/commits/f69475dc51aa4c9d4c43c0e089db5d8e254a5250) if the python check is not done.